src/ripple/app/paths/impl/PaySteps.cpp

```diff
@@ -390,11 +390,11 @@ toStrandV2 (
         !isConsistent(deliver) || (sendMaxIssue && !isConsistent(*sendMaxIssue)))
         return {temBAD_PATH, Strand{}};
 
-    for (auto const& p : path)
+    for (auto const& pe : path)
     {
-        auto const t = p.getNodeType();
+        auto const t = pe.getNodeType();
 
-        if (t > STPathElement::Type::typeAll || !t)
+        if (t & ~STPathElement::Type::typeAll)
```

Comment:
Previously there was a check here for `t` having no bits set.  That check seemed useful and is not covered by the new check.  Was the no-bits check lost intentionally?